### DISCUSSION TITLE
WebGl2

### DISCUSSION THREAD

**Discussion by aemonm** [uber/react-map-gl#2267] [General]
Any plans to support webgl2 in the near future ? I have a project that uses interleaved mode and it would be great to also have webgl2 for the rendering context.

**Comment by Pessimistress** [comment]
This library is a wrapper of mapbox-gl/maplibre-gl. It does not do any rendering itself. You need to ask in the Mapbox GL JS repo.

Maplibre is already using webgl2 in the latest release.
